node-data/src/ledger/transaction.rs

```diff
@@ -90,6 +97,24 @@ impl PartialEq<Self> for SpentTransaction {
 
 impl Eq for SpentTransaction {}
 
+pub enum SpendingId {
+    Nullifier([u8; 32]),
```

Comment:
In this implementation the `SpendingId` for phoenix would be only one nullifier. We can however have up to 4 in a transaction and the previous implementation of `to_nullifiers` also returned a `Vec<[u8; 32]>` (one nullifier is 32 bytes).

Reply:
Transaction::to_spending_ids indeed return a Vec<SpendingId>

So the idea is that a PhoenixTx has 4SpendingId to check, and MoonlightTx only one